tests/test_extract.py

```diff
@@ -88,20 +88,20 @@ def test_selection():
     parse_json_lines(lines, model)
 
     # Test select
-    ic = model.index(1, 0, QModelIndex())
+    ic = model.sibling(1, 0, QModelIndex())
```

Comment:
`index` is not the same as `sibling`.

Reply:
Reverted those to the original `index`. My bad changed those by mistake, Thanks for pointing it out.